.eslintrc.js

```diff
@@ -17,11 +17,314 @@ module.exports = {
         'plugin:react/recommended', // Uses the recommended rules from @eslint-plugin-react
         'plugin:@typescript-eslint/recommended' // Uses the recommended rules from @typescript-eslint/eslint-plugin
     ],
-    plugins: ['prettier'],
+    plugins: [
+        'import',
+        'prettier',
+        'react',
+        'react-hooks',
+        '@typescript-eslint',
+        'arca'
+    ],
     rules: {
         'react/prop-types': 0,
         '@typescript-eslint/no-inferrable-types': 0,
         '@typescript-eslint/no-explicit-any': 0,
-        'prettier/prettier': 'error'
+        'prettier/prettier': 'error',
```

Comment:
IMO we should add the comment saying from where this rule set is coming from, to keep it in sync with upstream rule set changes.